cli/src/commands/deployments/describe.rs

```diff
@@ -97,7 +96,7 @@ async fn describe(env: &CliEnv, opts: &Describe) -> Result<()> {
     c_println!();
 
     c_title!("🤖", "Services");
-    for svc in deployment.services {
+    for svc in deployment.components {
         let Some(latest_svc) = latest_services.get(&svc.name) else {
```

Comment:
nit: Renaming (service -> component) seems to be not complete. Won't mention it anymore.

Reply:
I'm tracking this here https://github.com/restatedev/restate/issues/1225